tools/sidekick/config.json

```diff
@@ -7,36 +7,58 @@
       "environments": [ "edit" ],
       "isPalette": true,
       "paletteRect": "top: auto; bottom: 20px; left: 20px; height: 398px; width: 360px;",
-      "url": "https://main--milo--adobecom.hlx.page/tools/library",
+      "url": "https://milo.adobe.com/tools/library",
       "includePaths": [ "**.docx**" ]
-    },    
+    },   
     {
-      "id": "translate",
-      "title": "Translate",
+      "id": "tools",
+      "title": "Tools",
+      "isContainer": true
+    }, 
+    {
+      "containerId": "tools",
+      "id": "localize",
+      "title": "Localize",
       "environments": [ "edit" ],
-      "url": "https://milo.adobe.com/tools/translation/index.html",
+      "url": "https://main--milo--adobecom.hlx.page/tools/loc/index.html?project=cc--adobecom",
```

Comment:
Shouldn't this be loaded from `https://milo.adobe.com`, analogous to the other tools available?

Reply:
I see path to staged content in all milo consuming teams (dc, bacom) for localize.